app/main.c

```diff
@@ -629,7 +629,10 @@ void main(void)
             display_pass_count(pass_num);
             if (error_count == 0) {
                 display_status("Pass   ");
-                display_notice("** Pass completed, no errors **");
+                if (!big_status_displayed) {
```

Comment:
This condition could be handled inside display_big_status(). Then big_status_displayed wouldn't need to be globally visible.

Reply:
Right. Will be changed before merge.